<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fixed Asset Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="fixedAssetTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="value">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userIDLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="userPasswordLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Depreciation Period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depreciationPeriod">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userIDLabel_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchase Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="purchaseDate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="terminateButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depreciationRatio">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userPasswordLabel_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Depreciation Ratio</string>
    </property>
   </widget>
   <widget class="QLabel" name="userPasswordLabel_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="userPasswordLabel_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>years</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
